src/cloudai/report_generator/comparison_report.py

```
@@ -141,37 +272,68 @@ def get_bokeh_html(self) -> tuple[str, str]:
+            loader=jinja2.FileSystemLoader(self.template_path),
+            autoescape=jinja2.select_autoescape(),
+        )
+        rendered_reports = [
+            ("Comparison report", self.report_file_name, self._render_html(env, sections, console_html)),
+            ("Comparison report v2", self.report_file_name_v2, self._render_html_v2(env, sections)),
+        ]
+
+        for report_name, file_name, content in rendered_reports:
+            report_path = self.results_root / file_name
+            report_path.write_text(content)
+            logging.info(f"{report_name} created: {report_path}")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)